build/localnet/Tiltfile

```diff
@@ -120,8 +120,11 @@ docker_build_with_restart(
     "client-image",
     root_dir,
     dockerfile_contents="""FROM debian:bullseye
-RUN apt-get update && apt-get install -y procps
-WORKDIR /
+RUN apt-get update && apt-get install -y procps bash-completion jq
+RUN echo "source /etc/bash_completion" >> ~/.bashrc
+# tail -n +2 removes the first line of the completion script since the CLI spits out some logs
+RUN echo "source <(p1 completion bash | tail -n +2)" >> ~/.bashrc
+WORKDIR /root
```

Comment:
Noticed we're changing the WORKDIR.

I know it's likely harmless but want to make sure that:

1. You're sure there's no harm
2. It was tested/validated E2E

Reply:
@okdas Can you TAL?

Reply:
<img width="1359" alt="Screenshot 2023-06-30 at 18 43 35" src="https://github.com/pokt-network/pocket/assets/95193764/4af7c176-7475-4ef8-8985-f11743a9b6e2">

yup we should be GTG, not sure why workdir was set to `/`

Reply:
Works for me, feel free to merge!